app/build.gradle

```diff
@@ -43,32 +43,35 @@ android {
         debug {
             shrinkResources false
             minifyEnabled false
-            applicationIdSuffix ".debug"
+            applicationIdSuffix ".fenix.debug"
             manifestPlaceholders.isRaptorEnabled = "true"
             resValue "bool", "IS_DEBUG", "true"
             pseudoLocalesEnabled true
         }
         forPerformanceTest releaseTemplate >> { // the ">>" concatenates the raptor-specific options with the template
             manifestPlaceholders.isRaptorEnabled = "true"
-            applicationIdSuffix ".performancetest"
+            applicationIdSuffix ".fenix.performancetest"
             debuggable true
         }
         fenixNightlyLegacy releaseTemplate >> {
+            applicationIdSuffix ".fenix"
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
         }
         fenixNightly releaseTemplate >> {
-            applicationIdSuffix ".nightly"
+            applicationIdSuffix ".fenix.nightly"
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
         }
         fenixBeta releaseTemplate >> {
-            applicationIdSuffix ".beta"
+            applicationIdSuffix ".fenix.beta"
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
         }
         fenixProduction releaseTemplate >> {
+            applicationIdSuffix ".fenix"
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
         }
         fennecProduction releaseTemplate >> {
             buildConfigField "boolean", "USE_RELEASE_VERSIONING", "true"
+            applicationIdSuffix ".firefox"
```

Comment:
Are we not going to pave over Firefox Beta (`org.mozilla.firefox_beta`) and Fennec Nightly (`org.mozilla.fennec_aurora`)?  Not yet?

Reply:
Not decided yet. There are multiple scenarios being discussed. Since we only get a limited amount of tries, I'd definitely like to try that with other channels first. :)